go.work

```diff
@@ -0,0 +1,17 @@
+go 1.18
+
+use (
+	./batch-submitter
+	./bss-core
+	./gas-oracle
+	./indexer
+	./l2geth
+	./l2geth-exporter
+	./op-exporter
+	./proxyd
+	./teleportr
+)
+
+replace github.com/ethereum-optimism/optimism/bss-core v0.0.0 => ./bss-core
+
+replace github.com/ethereum-optimism/optimism/l2geth v0.0.0 => ./l2geth
```

Comment:
beautiful

Reply:
We can remove the replace statements once the go modules are in their new place (i.e. this PR is merged) and we run `go mod tidy`. The workspace will pull in the local version during development, and the `go mod` files will each define the actual go version to use when running go outside of the workspace.